package.json

```diff
@@ -69,11 +69,12 @@
     "@rollup/plugin-typescript": "8.3.0",
     "@rollup/pluginutils": "4.1.2",
     "@types/chrome": "0.0.176",
+    "@types/eslint": "^8.4.1",
```

Comment:
We use exact version(please remove the `^`)

Reply:
Done